.rubocop.yml

```diff
@@ -41,6 +41,8 @@ Metrics/MethodLength:
   CountComments: false
   Max: 20
   Severity: error
+  Exclude:
+    - lib/jekyll/command.rb
```

Comment:
If that's not acceptable, there's another hack. `c.option "source"` is technically a duplicate as the `jekyll` executable already adds that option **to all Jekyll Commands**. So you can remove that one line from here.. again, just my 2c